wire-ios/Wire-iOS/Sources/SwiftUI Helpers/HapticFeedbackModifier.swift

```diff
@@ -0,0 +1,56 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+
+struct HapticTapGestureViewModifier: ViewModifier {
+    var style: UINotificationFeedbackGenerator.FeedbackType
+
+    func body(content: Content) -> some View {
+        content.onTapGesture {
+            let feedback = UINotificationFeedbackGenerator()
+            feedback.prepare()
+            feedback.notificationOccurred(style)
+        }
+    }
+}
+
+extension View {
+    func hapticFeedback(_ feedbackType: UINotificationFeedbackGenerator.FeedbackType) -> some View {
+        self.modifier(HapticTapGestureViewModifier(style: feedbackType))
+    }
+}
```

Comment:
This isn't being used anymore, perhaps we should delete it.

Reply:
Done!